lib/metadata/wrapper.js

```diff
@@ -117,9 +117,9 @@ const metadata = {
         });
     },
 
-    listObject: (bucketName, prefix, marker, delimiter, maxKeys, log, cb) => {
+    listObject: (bucketName, listingParams, log, cb) => {
```

Comment:
i'm on @ttvi side, options should always be an object, to avoid modify everything when we add an option :)